src/streamlink/plugins/n13tv.py

```diff
@@ -0,0 +1,117 @@
+import logging
+import re
+
+from streamlink.compat import urlparse
+from streamlink.plugin import Plugin
+from streamlink.plugin.api import validate
+from streamlink.stream import HLSStream
+
+log = logging.getLogger(__name__)
+
+
+class N13TV(Plugin):
```

Comment:
what does the `N` stand for?

Reply:
The N is for network.  Network 13 is the organisation that runs 13tv, according to their website.  I did wonder about calling this plugin thirteentv or perhaps thirteentv-co-il or similar.  I'm not sure if there are any particular preferences for plugin naming...

Thanks for all the code suggestions.  That'll keep me busy!